cardano-crypto-tests/vector-tests/TestVector/SchnorrSecp256k1Vectors.hs

```diff
@@ -0,0 +1,185 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Yeah it can be genarlized further i was trying to write independent tests according to Ecdsa or Secp but it is great with Proxy:: approach. Thanks will be updated accordingly.